### PR TITLE
Connect the Book Store to an external API

### PR DESCRIPTION
**Pull request by henrycode460**
This branch will do the followings:

- users will add book to the api
- books added to the API will be retrieve and display in the dom
- user can delete book from the api

### CODE REVIEW

File: src/Components/BookList.js [hunk 1]
```diff
@@ -1,16 +1,24 @@
...
+      {Object.keys(books).map((book) => (
+        <Book
+          key={book}
+          bookId={book}
+          title={books[book][0].title}
+          author={books[book][0].author}
+        />
+      ))}
+    </ul>
+  </>
```
Comment (ahmetbozaci) [inline src/Components/BookList.js:16]: - [ ] After add the book ı am getting an error. The book is not shown directly. I have to refresh the page to see the book that ı add.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @henrycode460  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] Home link in the navbar should open the main page, not the empty page. This has to done on the last milestone.

image

@henrycode460 
⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 
- You have one commit for everything. For best practices, you should commit often. [Git Commit Best Practices](https://www.perforce.com/blog/vcs/git-best-practices-git-commit)
⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by henrycode460** [comment]
Can we meet on zoom for 10 minutes please

**Review by mateo951 — APPROVED** [review]
### Hi @henrycode460, 

Well done making the necessary changes requested by the previous reviewer 🤓
Your project is complete! There is nothing else to say other than... it's time to merge it 🚀 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**